brownie-config.yml

```diff
@@ -2,21 +2,22 @@
 # NOTE: You don't *have* to do this, but it is often helpful for testing
 networks:
   default: mainnet-fork
-
+  kovan:
+    host: https://kovan.infura.io/v3/$WEB3_INFURA_PROJECT_ID
 # automatically fetch contract sources from Etherscan
 autofetch_sources: True
 
 # require OpenZepplin Contracts
 dependencies:
-  - iearn-finance/yearn-vaults@0.3.0
+  - iearn-finance/yearn-vaults@0.3.1
```

Comment:
Should handle this separately (maybe rebase on this change)

Reply:
Was trying deployment script sing a 0.3.1 vault so that's why I decided to update it too.